cmake/O2Dependencies.cmake

```diff
@@ -272,15 +272,6 @@ o2_define_bucket(
     ${CMAKE_SOURCE_DIR}/Utilities/PCG/include
 )
 
-o2_define_bucket(
-    NAME
-    O2FrameworkCore_benchmark_bucket
-
-    DEPENDENCIES
-    O2FrameworkCore_bucket
-    $<IF:$<BOOL:${benchmark_FOUND}>,benchmark::benchmark,$<0:"">>
-)
```

Comment:
why did you remove this? Seems not related to this PR and the compilation seems to fails due to this

Reply:
My mistake. I have fixed it. (Change not yet committed.)